web/src/components/netflow-traffic.tsx

```diff
@@ -49,18 +92,29 @@ const NetflowTraffic: React.FC = () => {
       })
   }, [false /*temp: no refresh*/]);
 
-    return !_.isEmpty(extensions) ? (
-	<PageSection>
-	    <h2>Network Traffic</h2>
-	    {loading && <>Loading...</>}
-	    {error && (
-		<div>Error: {error}</div>
-	    )}
-	{!_.isEmpty(flows) &&
-	 <NetflowTable flows={flows} setFlows={setFlows} columns={Columns} />
-	}
-	</PageSection>
-    ) : null;
+  return !_.isEmpty(extensions) ? (
+    <PageSection id="pageSection">
+      <h2>Network Traffic</h2>
+      {loading && <>Loading...</>}
+      <Toolbar id="filter-toolbar">
+        <ToolbarContent>{toolbarItems}</ToolbarContent>
+      </Toolbar>
+      {error && (
+        <div>Error: {error}</div>
+      )}
+      {!_.isEmpty(flows) &&
+        <NetflowTable
+          flows={flows}
+          setFlows={setFlows}
+          columns={columns.filter((col) => col.isSelected)} />
+      }
+      <ColumnsModal
+        id="columns-modal"
+        isModalOpen={isModalOpen}
+        setModalOpen={setModalOpen}
+        columns={columns}
+        setColumns={setColumns} />
+    </PageSection>) : null;
```

Comment:
looks like there's an indentation issue here. Not that I care for this PR, but we should have an auto-formatter

Reply:
actually, my bad, I didn't read correctly. though we still should have automatic formatting :)

Reply:
yep, should we create [vscode workspace settings](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations) or do you have something better in mind ?

I guess this should be done on all projects

Reply:
I think there's equivalent things that are more standard, not vscode specific. IIRC @OlivierCazade doesn't use vscode, right?
Also it could be precommit hooks or such with something like [prettier](https://prettier.io/) but, to be clear, I don't like this one in particular (too opinionated IMO , and slow iirc)

Reply:
Yes, if we could find a formatting that is not specific to an editor it would be much appreciated.